index.ts

```diff
@@ -123,22 +138,40 @@ app.all("/stimmung/:room?", checkAuth, function (req, res) {
       : "",
     html_author: process.env.HTML_AUTHOR ? process.env.HTML_AUTHOR : "",
     name: req.session.name,
-    room: req.params.room || '',
+    room: req.params.room || "",
   });
 });
 
 server.listen(process.env.PORT, () => {
   console.log(`Server listening on port ${process.env.PORT}`);
 });
 
-var WSS = require("websocket").server;
+import * as websocket from "websocket";
```

Comment:
Moving all `import`s to the top as in the previous draft is actually a good idea.